core/translations/fr.json

```diff
@@ -276,7 +275,7 @@
     "coinjoin__title_progress": "Coinjoin en cours",
     "coinjoin__waiting_for_others": "En attente des autres",
     "confirm_total__fee_rate": "Taux des frais",
-    "confirm_total__fee_rate_colon": "Taux des frais :",
+    "confirm_total__fee_rate_colon": "Taux des frais :",
```

Comment:
this should be also removed I suppose

Reply:
Ideally yes, but this is used from Rust and we don't currently seem to have a way to format strings there.

Reply:
We use heapless. All allocated on the stack. Very limited functionality. Not saying it is not doable, but so far I did not see any string formatting in our Rust code.

Reply:
FWIW we have [ufmt](https://docs.rs/ufmt/latest/ufmt/) but it requires the formatting string to be known at compile-time.

This could be removed by formatting the string in python before passing it as a new parameter to the layout.